static/app/views/settings/account/notifications/notificationSettingsByType.spec.tsx

```diff
@@ -34,97 +34,81 @@ const addMockResponses = (
     method: 'GET',
     body: [],
   });
-};
+}
 
-const createWrapper = (
+function TestComponent(
```

Comment:
I've been calling this `renderComponent`. `TestComponent` with the capitalization implies this is a react component, not a plain function

Reply:
yes it makes sense. will update it :) we will have to do a cleanup later, I guess